gelfclient/client.py

```diff
@@ -44,6 +50,9 @@ def log(self, _fields_dict = {}, **fields_named):
                 message['host'] = message['source']
             else:
                 message['host'] = self.source
+        if 'level' not in message:
+            message['level'] = LogLevel.alert
```

Comment:
Considering my other request to remove the LogLevel class, please change this to ```message['level'] = 1``` then we don't need to import the syslog module.

Another alternative would be to not set any default level value at all, since the spec says it is optional and I'm guessing it will be set to 1 on the Graylog server-side